iron-multi-selectable.html

```diff
@@ -46,7 +46,7 @@
     },
 
     observers: [
-      '_updateSelected(selectedValues)'
+      '_updateSelected(selectedValues.*)'
```

Comment:
@andrewmwhite After updating, can you also sign the CLA so we can land this? Thanks.